mage/query-modules/python/gnn-node-classification.md

```diff
@@ -0,0 +1,342 @@
+---
+id: gnn-node-classification
+title: gnn-node-classification
+sidebar_label: gnn-node-classification
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+{children}
+</span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-node_classification-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/T515-IN-node-classification-with-pyg/python/node_classification.py)
+
+## Abstract
+
+Intuitively, node classification is problem where one has to determine labelling of samples from other neighbours’ labels.
+Little more formally, let $G=(V,E)$ be a graph, S set of all labels, and $f : V → S$ vertex labelling function which returns true labels. From given partial function $g : V’ ⊆ V → S$ (training examples) node classification model aims to learn function $g’ : V → S$ which tries to predict as many true labels as possible. It is motivated by sociological concept of “homophily”: individual nodes tend to associate and bond with each other because they probably have similar labels.
+
+## User story
+
+Set $V$ in this case can be a lot of things, the most common are scientific publications, proteins, Reddit posts etc. In the same respective order, set $E$ can be citation of publications, protein connections or connection of posts commented by same user. Therefore, if user wants to classify publication relevant to its field, find if protein is helpful to curing new diseases, or simply find out to which subreddit its post naturally belongs, it should try out this approach. Given any not completely labeled graph, this problem will find all nodes’ labels.
+
+In this query module, we will concentrate on one very important use case of node classification: **Fraud detection**.
```

Comment:
It seems to me that this module doesn't have any features specific to fraud detection.
I’d cut this sentence, and instead edit the user story so that it's about fraud detection and not science/reddit datasets